erpnext/stock/doctype/material_request/material_request.py

```
@@ -887,6 +887,16 @@ def raise_work_orders(material_request):
 
 @frappe.whitelist()
 def create_pick_list(source_name, target_doc=None):
+	def update_item(obj, target, source_parent):
```

Ruff (0.14.3)

Unused function argument: `source_parent`

(ARG001)